roles/kubernetes-apps/scheduler_plugins/templates/controller-deploy.yml.j2

```diff
@@ -0,0 +1,22 @@
+kind: Deployment
```

Comment:
the `registry.k8s.io/scheduler-plugins/kube-scheduler` will replace the default kube-scheduler's image .
But how about `registry.k8s.io/scheduler-plugins/controller`?
it will be another pod running side by side with kube-controller, or will replace default kube-controller image?

Reply:
The pod that declare its image as `registry.k8s.io/scheduler-plugins/controller` is the controller to handle the CRDs of `scheduler-plugins` only. So the controller pod would not replace default kube-controller image.